bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/Znunugamma_aTGC/Znunugamma_aTGC_ZZg_h3Z_h4Z_Ptg150to600_2jets/Znunugamma_aTGC_ZZg_h3Z_h4Z_Ptg150to600_2jets_run_card.dat

```diff
@@ -0,0 +1,253 @@
+  T	=  auto_ptj_mjj   ! Automatic setting of ptj and mjj
+#**********************************************************
+#                                                                    
+#**********************************
+# BW cutoff (M+/-bwcutoff*Gamma)
+#**********************************
+  15	=  bwcutoff       ! (M+/-bwcutoff*Gamma)
+#**********************************************************
+# Apply pt/E/eta/dr/mij cuts on decay products or not
+# (note that etmiss/ptll/ptheavy/ht/sorted cuts always apply)
+#**********************************************************
+  F	=  cut_decays     ! Cut decay products 
+#*************************************************************
+# Number of helicities to sum per event (0 = all helicities)
+# 0 gives more stable result, but longer run time (needed for
+# long decay chains e.g.).
+# Use >=2 if most helicities contribute, e.g. pure QCD.
+#*************************************************************
+  0	=  nhel           ! Number of helicities used per event
+#*******************                                                 
+# Standard Cuts
+#*******************                                                 
+#                                                                    
+#*********************************************************************
+# Minimum and maximum pt's (for max, -1 means no cut)                *
+#*********************************************************************
+  10	=  ptj        ! minimum pt for the jets 
+  0	=  ptb        ! minimum pt for the b 
+  150	=  pta        ! minimum pt for the photons 
+  0	=  ptl        ! minimum pt for the charged leptons 
+  0	=  misset     ! minimum missing Et (sum of neutrino's momenta)
+  0	=  ptheavy    ! minimum pt for one heavy final state
+  1.0	=  ptonium    ! minimum pt for the quarkonium states
+  -1	=  ptjmax     ! maximum pt for the jets
+  -1	=  ptbmax     ! maximum pt for the b
+  600	=  ptamax     ! maximum pt for the photons
+  -1	=  ptlmax     ! maximum pt for the charged leptons
+  -1	=  missetmax  ! maximum missing Et (sum of neutrino's momenta)
+#*********************************************************************
+# Minimum and maximum E's (in the center of mass frame)              *
+#*********************************************************************
+  0	=  ej      ! minimum E for the jets 
+  0	=  eb      ! minimum E for the b 
+  0	=  ea      ! minimum E for the photons 
+  0	=  el      ! minimum E for the charged leptons 
+  -1	=  ejmax  ! maximum E for the jets
+  -1	=  ebmax  ! maximum E for the b
+  -1	=  eamax  ! maximum E for the photons
+  -1	=  elmax  ! maximum E for the charged leptons
+#*********************************************************************
+# Maximum and minimum absolute rapidity (for max, -1 means no cut)   *
+#*********************************************************************
+  5	=  etaj     ! max rap for the jets 
+  -1	=  etab     ! max rap for the b
```

Comment:
please synchronize j and b related cuts. Otherwise it can create some weird distortions in phasespace

Reply:
Changed to  5	=  etab     ! max rap for the b